woocommerce/woocommerce-android
# Use generic currency symbol

Parent issue https://github.com/woocommerce/woocommerce-android/issues/4553


### Description

This PR removes hardcoded "$" currency symbol and uses a utility class to get a currency symbol at runtime.

It also removes CurrencyModule as it isn't necessary - we can use @Inject annotation on a constructor instead.

### Testing instructions

- Make sure you are using a store which is eligible for in-person payments
1. Open detail of an unpaid order in USD
2. Tap on Collect Payment
3. Connect to a reader
4. Wait until the app prompts you to insert/tap/swipe your card
5. Notice the currency symbol at the top of the dialog is "$"


### Images/gif
